plugins/grpc-plugin/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+edition = "2021"
+name = "grpc-plugin"
+version = "0.1.0"
```

Comment:
maybe it is missing a space here between the two sections?

Reply:
Fixed up in 56d1820c9